src/Helper/AbstractList.php

```diff
@@ -46,14 +46,13 @@ abstract class AbstractList extends AbstractHelper
 
     /**
      *
-     * Initializes and returns the UL object.
+     * Initializes and returns the list object.
      *
-     * @param array $attr Attributes for the UL tag.
+     * @param array<string, scalar|null>|null $attr HTML element attributes
      *
      * @return self
-     *
      */
-    public function __invoke(array $attr = null)
+    public function __invoke($attr = null)
```

Comment:
If we change the signature then it is a breaking change. I will be happy to merge if we don't break. I am checking from mobile. So limited files are checked. I will look more closely from desktop and let you know. Also thanks for the reminder.

Reply:
> I checked and the class isn't final, so this could break child classes that override with stricter types.

This was my misunderstanding. Changing `array $attr` to `?array $attr` or `null|array $attr` does not constitute a breaking change.